### PR TITLE
Meetup events

### PR DESCRIPTION
**Pull request by Guitarbum722**
* Add Methods to query Event data
* Add tests with new mocks

There is quite a bit of functionality centered around events (obviously).  I will add more functionality such as the creation, updates and comments of events in one or two separate PRs.

### CODE REVIEW

File: event_constants.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package meetup
+
```
Comment (briandowns) [inline event_constants.go]: This file can be removed and the constants be added to the top of events.go.

File: events.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	v := c.urlValues()
+	if radius == "" {
+		radius = smartRadius
+	}
+	v.Add("radius", radius)
+	v.Add("lon", lon)
+	v.Add("lat", lat)
+
+	uri := eventsConcierge + queryStart + v.Encode()
+	fmt.Println(uri)
```
Comment (briandowns) [inline events.go]: Permanent?

File: models/event.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package models
+
+// Event describes a meetup event and pertinent information such as id, comments, etc.
+type Event struct {
+	ID       string `json:"id"`
+	Name     string `json:"name"`
+	Link     string `json:"link"`
+	YesRSVP  int    `json:"yes_rsvp_count"`
+	Waitlist int    `json:"waitlist_count"`
+	Group    `json:"group"`
```
Comment (briandowns) [inline models/event.go]: Missing type?

Comment (Guitarbum722) [inline models/event.go]: Group is of type `Group`.  Is it still better to specify if it is the same name?

File: events.go (hunk 2 above)
Comment (Guitarbum722) [inline events.go]: Not permanent 😆

File: models/event.go (hunk 3 above)
Comment (Guitarbum722) [inline models/event.go]: fixed

File: event_constants.go (hunk 1 above)
Comment (Guitarbum722) [inline event_constants.go]: done

**Comment by Guitarbum722** [comment]
Thank you, @briandowns !

**Comment by briandowns** [comment]
You're welcome! :D

**Pull request merged**